### PR TITLE
feat(issues): add github issue template for v2 draft feedback

### PR DESCRIPTION
**Pull request by kburger**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feedback_template.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    attributes:
+      label: "Proposed change"
+      description: "Please describe your proposed adjustment or addition to the metadata model"
+      placeholder: ""
+
+  - type: dropdown
+    id: priority
+    attributes:
+      label: "Priority"
+      description: "Indicate/choose the priority level of your issue"
```
Comment (kburger) [inline .github/ISSUE_TEMPLATE/feedback_template.yml:60]: The wording here could be improved, open for suggestions

File: .github/ISSUE_TEMPLATE/feedback_template.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+description: "Metadata model draft version 2 feedback form"
+title: "feedback: "
+labels: ["draft-v2-feedback"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Health-RI metadata model version 2 draft feedback
+
+        Please use this form to describe feedback related to the plateau 2 draft version of the core / generic health metadata schema (link).
```
Comment (kburger) [inline .github/ISSUE_TEMPLATE/feedback_template.yml:11]: Link target needs to be decided, and properly encoded in markdown.

File: .github/ISSUE_TEMPLATE/feedback_template.yml (hunk 1 above)
Comment (HNeikes) [reply .github/ISSUE_TEMPLATE/feedback_template.yml:60]: Suggestion: "Please indicate the priority level of your issue."

File: .github/ISSUE_TEMPLATE/feedback_template.yml (hunk 2 above)
Comment (HNeikes) [reply .github/ISSUE_TEMPLATE/feedback_template.yml:11]: Based on the discussion yesterday, link will be to develop branch.

https://github.com/Health-RI/health-ri-metadata/tree/develop

**Review by HNeikes — APPROVED** [review]

**Pull request merged**